src/System.ComponentModel.Primitives/ref/System.ComponentModel.Primitives.cs

```diff
@@ -51,28 +51,28 @@ public partial class Component : System.MarshalByRefObject, System.ComponentMode
         protected virtual bool CanRaiseEvents { get { throw null; } }
         [System.ComponentModel.BrowsableAttribute(false)]
         [System.ComponentModel.DesignerSerializationVisibilityAttribute(System.ComponentModel.DesignerSerializationVisibility.Hidden)]
-        public System.ComponentModel.IContainer Container { get { throw null; } }
+        public System.ComponentModel.IContainer? Container { get { throw null; } }
         [System.ComponentModel.BrowsableAttribute(false)]
         [System.ComponentModel.DesignerSerializationVisibilityAttribute(System.ComponentModel.DesignerSerializationVisibility.Hidden)]
         protected bool DesignMode { get { throw null; } }
         protected System.ComponentModel.EventHandlerList Events { get { throw null; } }
         [System.ComponentModel.BrowsableAttribute(false)]
         [System.ComponentModel.DesignerSerializationVisibilityAttribute(System.ComponentModel.DesignerSerializationVisibility.Hidden)]
-        public virtual System.ComponentModel.ISite Site { get { throw null; } set { } }
+        public virtual System.ComponentModel.ISite? Site { get { throw null; } set { } }
         [System.ComponentModel.BrowsableAttribute(false)]
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Advanced)]
-        public event System.EventHandler Disposed { add { } remove { } }
+        public event System.EventHandler? Disposed { add { } remove { } }
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
         ~Component() { }
-        protected virtual object GetService(System.Type service) { throw null; }
-        public override string ToString() { throw null; }
+        protected virtual object? GetService(System.Type service) { throw null; }
+        public override string? ToString() { throw null; }
```

Comment:
I'm curious why you made this nullable.  Do you know of a case where a Component-derived type returns null from ToString?

Reply:
Just because of Type.FullName, got feedback from Maryam and Santi about that https://github.com/dotnet/corefx/pull/41185#discussion_r325916170 and made it not nullable, but seems I didn't pushed that change yet